src/main/resources/hudson/plugins/batch_task/Messages.properties

```diff
@@ -6,3 +6,4 @@ BatchTaskInvoker.NoSuchProject=No such project exists: {0}
 BatchTaskInvoker.NoSuchTask=No such task exists: {0}. Perhaps you meant {1}
 BatchTaskAction.DisplayName={0,choice,1#Task|1<Tasks}
 BatchRunAction.DisplayName=Executed Tasks
+BatchTaskInvoker.ForwardSlash=Project should be a Full Project Name which cannot start by /
```

Comment:
It would be useful to clarify the impact of such misconfiguration. "Current settings may lead to the undefined behavior" seems to be scary enough for me :)